oap-stdlib/src/test/java/oap/application/KernelProfileTest.java

```diff
@@ -53,11 +51,15 @@ public void serviceProfiles() {
 
         try( var kernel = new Kernel( modules ) ) {
             startWithProfile( kernel, "test-profile", "profile-name1" );
+            assertThat( kernel.<TestProfile1>service( "*.profile" ) ).isNotPresent();
+        }
+        try( var kernel = new Kernel( modules ) ) {
+            startWithProfile( kernel, "test-profile", "profile-name1", "profile-name2" );
```

Comment:
Could you clarify why exciting test edited?

Reply:
Git shows it with some confusion. I've added a block for checking disabled profiles. 
```
try( var kernel = new Kernel( modules ) ) {
            startWithProfile( kernel, "test-profile", "profile-name1", "profile-name2" );
            assertThat( kernel.<TestProfile1>service( "*.disabledProfile" ) ).isNotPresent();
        }
```